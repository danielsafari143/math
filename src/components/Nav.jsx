import { Link } from 'react-router-dom';

const Nav = () => (
  <div className="nave">
    <h1>Math Magicians</h1>
    <nav>
      <Link className="link" to="/">Home</Link>
      <Link className="link" to="calculator">Calculator</Link>
      <Link className="link" to="quote">Quote</Link>
    </nav>
  </div>
);

export default Nav;
