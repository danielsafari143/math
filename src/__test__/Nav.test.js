import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Nav from '../components/Nav';

describe('navbar', () => {
  it('Test if the UI matches ', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Nav />
      </MemoryRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
