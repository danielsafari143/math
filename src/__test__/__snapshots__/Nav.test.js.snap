// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar Test if the UI matches  1`] = `
<div
  className="nave"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <a
      className="link"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="link"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="link"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;
